blockdag/dagio.go

```diff
@@ -302,85 +304,84 @@ func (dag *BlockDAG) initDAGState() error {
 	}
 
 	// Attempt to load the DAG state from the database.
-	return dag.db.View(func(dbTx database.Tx) error {
-		// Initialize the reachability store
-		log.Infof("Loading reachability data...")
-		err = dag.reachabilityStore.init(dbaccess.NoTx())
+
+	// Initialize the reachability store
```

Comment:
IMHO these comments are redundant (especially that we have a log that says the same thing right below)

Reply:
On second thought, there are other stages where the code isn't as clear, and there a comment is in place, so for the sake of uniformity might want to leave these comments as well - your call